misc/sysuser/linglong.conf

```diff
@@ -6,4 +6,4 @@
 # at boot on systemd-based systems that ship with an unpopulated /etc/.
 # See sysuser.d(5) for details.
 
-u linglong - "linglong Package Manager"
+u deepin-linglong - "linglong Package Manager"
```

Comment:
cc @Iceyer 

> Specify "-" for automatic UID/GID allocation for the user or group (this is strongly recommended unless it is strictly necessary to use a specific UID or GID).

[ref](https://www.freedesktop.org/software/systemd/man/sysusers.d.html#ID)

I removed the uid range limit.

Reply:
> which should be man

sysuser will keep it less then 1000, this should be fine.